src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -20,7 +25,11 @@ public Integer oddSum(List<Integer> numbers) {
      * Вернуть количество вхождений объекта `element`</p>
      **/
     public Long elementCount(List<String> elements, String element) {
-        return null;
+        return elements
+                .stream()
+                .filter(x -> x.equals(element))
+                .mapToInt(p -> 1)
```

Comment:
do we need to use this map?

Reply:
Here we transform String to 1 and then calculate their sum.

Reply:
Now I see thats it's obvious.